Source/HTTP/WinHttp/winhttp_provider.cpp

```diff
@@ -326,8 +327,23 @@ Result<XPlatSecurityInformation> WinHttpProvider::GetSecurityInformation(const c
 #endif
 }
 
-Result<HINTERNET> WinHttpProvider::GetHSession(uint32_t securityProtocolFlags)
+Result<HINTERNET> WinHttpProvider::GetHSession(uint32_t securityProtocolFlags, const char* url)
 {
+    // Parse URL to determine scheme
+    xbox::httpclient::Uri uri(url);
+    if (!uri.IsValid())
+    {
+        return E_INVALIDARG;
+    }
+    
+    bool isHttps = uri.IsSecure();
+    
+    // Log warning for insecure HTTP requests
+    if (!isHttps)
+    {
+        HC_TRACE_WARNING(HTTPCLIENT, "WARNING: Insecure HTTP request \"%s\"", url);
```

Comment:
For XCurl we had the policy to warn about HTTP use for historical and console certification reasons. I think it's fine to say that LibHttpClient does not have the same policy and where not strictly required (i.e. most platforms) you won't generate noise about HTTP usage.

I'd recommend either removing this *or* putting it behind some GDK ifdef *or* some GDK-Console runtime flag